alma/alma-woocommerce-gateway
# Hpos compatibility

## Reason for change

HPOS compatibility

[Task](https://linear.app/almapay/issue/ECOM-1381/%5B%F0%9F%A7%A9-wc%5D-fix-hpos-compatibility)

### How to test

- uninstall wmpl
- In the admin go to Pages -> checkout -> edit
- select the block 
![image](https://github.com/alma/alma-woocommerce-gateway/assets/110386752/b4dda8bd-339d-4757-8aff-60c1043758dd)
- click on switch to classic checkout
![image](https://github.com/alma/alma-woocommerce-gateway/assets/110386752/3c30f2e3-0b5a-4461-bad9-2ab18df4fea0)
- click on update in top of the page
- In Woocommerce admin : go to woocommerce -> settings -> advanced -> features
- We can now enabled High Performance order storage and compatibility mode
- Place alma orders
- Then do a full refund with alma, partial refund and check if you have the notice on the right and the notice in the page 
![image](https://github.com/alma/alma-woocommerce-gateway/assets/110386752/264a9d44-920c-4cf2-b43a-748231c92c2e)
